development.md

```diff
@@ -0,0 +1,69 @@
+# Development and contribution
+Prerequisite: Install git, node package manager, webpack CLI, grunt CLI, Karma CLI
```

Comment:
move into directory `docs` with the other widget readme files
Always keep the main readme in the root, for GitHub to pick it up as the main documentation